herbert_core/utilities/maths/replicate_array.m

```diff
@@ -3,6 +3,15 @@
 %
 %   >> vout = replicate_array (v, n)
 %
+% This is a legacy function retained for backwards compatibility. The same
+% behaviour is accomplished with the Matlab intrinsic function repelem which was
+% introduced in release R2015a:
+%  Instead of:
+%   >> vout = replicate_array (v, n)
+%  use:
+%   >> vout = repelem (v(:), n(:))
```

Comment:
basically all 3 replicate functions now have identical implementation. Guess ok as it's legacy back compatibility

Reply:
They are not quite identical: replicate_logarray has a type conversion as well. It is certainly my goal to use repelem in the future. However, there are several places where they are used around Horace and in my own code (although repelem was introduced in 2015a, I only learned of its existence a week or two ago.